app/src/main/java/org/akvo/flow/service/bootstrap/BootstrapProcessor.kt

```diff
@@ -85,15 +85,15 @@ class BootstrapProcessor @Inject constructor(
             // now read the survey XML back into memory to see if there is a version
             val surveyMetadata = fileProcessor.readBasicSurveyData(surveyFile)
 
-            //most instances use the same for server base and instance url now
-            val instanceCodeName: String = if (BuildConfig.SERVER_BASE == BuildConfig.INSTANCE_URL) {
-                BuildConfig.AWS_BUCKET
+            if (surveyMetadata.alias.isNotEmpty() ) {
+                if (!BuildConfig.INSTANCE_URL.contains(surveyMetadata.alias)) {
```

Comment:
Can this be passed incase there are two dashboards that contain similar alias even though the full string is not the same? e.g. `dashboard12` and `dashboard1`?

Reply:
no, for example, for uat1, INSANCE_URL="https://uat1.akvoflow.org" and alias would be "uat1.akvoflow.org", this would not work with uat2